go/obscuronode/host/host.go

```diff
@@ -580,7 +592,7 @@ func (a *Node) awaitSecret() {
 
 		case <-time.After(time.Second * 10):
 			// This will provide useful feedback if things are stuck (and in tests if any goroutines got stranded on this select
-			nodecommon.LogWithID(a.shortID, "Still waiting for secret from the L1...")
+			nodecommon.LogWithID(a.shortID, "Waiting for secret from the L1 after %s...", startTime)
```

Comment:
time you mean to put `time.Since(startTime)` as the argument here?

Reply:
Yes! Good catch ! 👍